src/ant/src/AntennaChecker.cc

```diff
@@ -756,6 +764,7 @@ void AntennaChecker::writeReport(std::ofstream& report_file, bool verbose)
 
 void AntennaChecker::printReport()
 {
+  std::lock_guard<std::mutex> lock(map_mutex_);
```

Comment:
Same question here. printReport is only called inside checkAntennas, out of a multithread region of the code.